src/background_process.c

```diff
@@ -164,6 +164,16 @@ void *background_process(void *ptr)
 		    log_to_disk(true);
 		    break;
 
+		case QTCRENTRY:
+
+		    store_recv_qtc(lan_message+2);
+		    break;
+
+		case QTCSENTRY:
+
+		    store_sent_qtc(lan_message+2);
+		    break;
+		    
```

Comment:
Hmm, that's very interesting. Of course, I've added header files to background_process.c.
But now I'm a littlebit confused: the source tree compiled succesfully, the compiler not misses this header (and prototypes). I think, because it found the Makefile. I've tested the network mode too, so there are two clients, and they communicates with each other without problem. And as I see, there is an another function in background_process.c, which not has prototype in the C source: qsonr_to_str(). I have to read about it... :)